src/shared/fs/watchedFiles.ts

```diff
@@ -221,7 +221,7 @@ export abstract class WatchedFiles<T> implements vscode.Disposable {
      * Rebuilds registry using current glob and exclusion patterns.
      * All functionality is currently internal to class, but can be made public if we want a manual "refresh" button
      */
-    private async rebuild(): Promise<void> {
+    public async rebuild(): Promise<void> {
```

Comment:
looks like this can stay private, for now
```suggestion
    private async rebuild(): Promise<void> {
```